labs/lab0/chapter0.md

```diff
@@ -112,7 +111,7 @@ in your OpenShift instance, if you open a browser. However, before we do that, w
 $ minishift ip
 192.168.???.??? 
 ```
-Ok, now we can check out the OpenShift console. Open Firefox from the Applications menu and navigate to `https://<ip>:8443/console/`(replace `<ip>` with the address from the last command). Once it loads (and you bypass the bad certificate error), you can log in to the console using the default `developer/developer` username/password.
+Ok, now we can check out the OpenShift console. Open Firefox from the Applications menu and navigate to `https://<ip>:8443/console/`(replace "<ip>" with the address from the last command). Once it loads (and you bypass the bad certificate error), you can log in to the console using the default `developer/developer` username/password.
```

Comment:
I don't know why you made this change. I fixed it a while back so that `<ip>` shows up properly in the displayed page and not just blank quotes.